.travis.yml

```diff
@@ -33,7 +35,13 @@ before_script:
   - composer install
 
   # Start a webserver for web fixtures. Force using PHP 5.6 to be able to run it on PHP 5.3 and HHVM jobs too
-  - MINK_PHP_BIN=~/.phpenv/versions/5.6/bin/php vendor/bin/mink-test-server > /dev/null 2>&1 &
+  - if [[ ${TRAVIS_PHP_VERSION:0:3} == "5.3" ]]; then MINK_PHP_BIN=~/.phpenv/versions/5.3/bin/php vendor/bin/mink-test-server > /dev/null 2>&1 & fi
```

Comment:
the mink test server cannot run on 5.3 (see the comment above)

Reply:
As you can see, I now have no idea what to do next, but am trying various different stuff to see what might be going on. The release of any new version of MinkSelenium2Driver seems to be stalled because for some reason the testing CI fails. If I could work out what the problem is and fix it, then happy to contribute.